joints:
   arm_1_joint:
      type: pal_local_joint_control::DirectPositionControl
   arm_2_joint:
      type: pal_local_joint_control::DirectPositionControl
   arm_3_joint:
      type: pal_local_joint_control::DirectPositionControl
   arm_4_joint:
      type: pal_local_joint_control::DirectPositionControl
   arm_5_joint:
      type: pal_local_joint_control::DirectPositionControl
   arm_6_joint:
      type: pal_local_joint_control::DirectPositionControl
   arm_7_joint:
      type: pal_local_joint_control::DirectPositionControl
   head_1_joint:
      type: pal_local_joint_control::DirectPositionControl
   head_2_joint:
      type: pal_local_joint_control::DirectPositionControl
   torso_lift_joint:
      type: pal_local_joint_control::DirectPositionControl

base_footprint:
   intialize_base_footprint_with_tf: false
